### PR TITLE
Add translation domains in install

### PR DESCRIPTION
**Pull request by Shine-neko**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Add translation domains in install |
| Type? | improvement |
| Category? | IN |
| BC breaks? | yes |
| Deprecations? | yes |

### CODE REVIEW

**Comment by rGaillard** [comment]
Without the intl extension this will crash before the requirements step, no?

**Comment by Quetzacoalt91** [comment]
capture du 2016-06-17 15-31-21

:boom:

**Comment by Shine-neko** [comment]
@Quetzacoalt91 I have not finished the pr ( wip powa) 
@rGaillard  I don't know. We must make the test

**Comment by mickaelandrieu** [comment]
@rGaillard @Shine-neko don't worry too much for intl support => https://github.com/symfony/symfony/blob/2.8/composer.json#L25

File: install-dev/controllers/http/configure.php [hunk 1]
```diff
@@ -30,13 +30,15 @@
 class InstallControllerHttpConfigure extends InstallControllerHttp
```
Comment (mickaelandrieu) [inline install-dev/controllers/http/configure.php]: could be worth to have ``trans()`` in InstallControllerHttp, no ?

**Pull request merged**